client/src/pages/DriverWaitingPage.tsx

```diff
@@ -61,10 +73,12 @@ function DriverWaitingPage() {
   }, [riderCalls]);
 
   useEffect(() => {
-    if (trip.id) {
-      getTripStatus(client, trip, setDriverStatus);
+    if (!!tripStatus && tripStatus.tripStatus === OPEN) {
```

Comment:
```suggestion
    if (!!tripStatus && tripStatus === OPEN) {
```
얘도 이렇게 되야하는게 아닌가요 혹시..?

Reply:
마찬가지로 데이터명을 헷갈리게 적은 것 같아서 수정했습니다!